app/views/admin/claims/show.html.erb

```diff
@@ -14,6 +14,12 @@
                   admin_claim_approvals_path(@claim),
                   method: :post,
                   class: "govuk-button",
-                  data: { confirm: "Are you sure you want to approve this claim?" } %>
+                  data: { confirm: "Are you sure you want to approve this claim?" },
+                  "data-module": "govuk-button" %>
+
+    <%= link_to "Reject", new_admin_claim_rejection_path(@claim),
+                class: "govuk-button govuk-button--warning",
+                "data-module": "govuk-button",
```

Comment:
Is this correct to do, given that it's an `<a>` not a `<button>`?

Reply:
My thinking is that as this behaves as a button, whatever JS is used on GOVUK buttons uses this to target buttons, therefore we need it?

I think it's this:

https://github.com/alphagov/govuk-frontend/blob/bf8bedf84f984fa4dabbda1d8f4081820d2e9e2b/src/govuk/all.js#L20

Reply:
Ah, I was under the false impression that that assumed it was a `<button>`, but I have dug further, and it looks like it supports other tags that use `role="button"`: https://github.com/alphagov/govuk-frontend/blob/bf8bedf84f984fa4dabbda1d8f4081820d2e9e2b/src/govuk/components/button/button.js#L25